form-builder/index.html

```diff
@@ -0,0 +1,1201 @@
+                                                required>
+                                                <option style="color:#545b62" value="" disabled selected>Select a
+                                                    question type...</option>
+                                                <option value="acknowledgement">Acknowledgement</option>
+                                                <option value="boolean">Boolean</option>
+                                                <option value="choice">Choice</option>
+                                                <option value="date">Date</option>
+                                                <option value="decimal">Decimal</option>
+                                                <option value="dispatch-job">Dispatch Job (ID)</option>
+                                                <option value="employee">Employee (ID)</option>
+                                                <option value="equipment">Equipment (ID)</option>
+                                                <!-- <option value="geolocation">Geolocation</option> -->
+                                                <option value="initials">Initials</option>
+                                                <option value="integer">Integer</option>
+                                                <option value="signature">Signature</option>
+                                                <option value="text">Text</option>
+                                                <option value="time">Time</option>
+                                            </select>
+                                        </div>
+
+                                        <div class="form-group form-check d-inline-block">
+                                            <label class="form-check-label">
+                                                Required
+                                                <input class="form-check-input" type="checkbox"
+                                                    data-bind="checked: required">
+                                            </label>
+                                        </div>
+
+                                        <!-- ko if: type() === 'text' -->
+                                        <div class="form-group form-check d-inline-block">
+                                            <label class="form-check-label">
+                                                Multi-Line Text
+                                                <input class="form-check-input" type="checkbox"
+                                                    data-bind="checked: multiLineText">
+                                            </label>
+                                        </div>
+                                        <!-- /ko -->
+
+                                        <!-- ko if: type() === 'choice' -->
+                                        <div class="form-group choices-field">
```

Comment:
The `Required` checkbox should display the same line as `Multi Choice` checkbox for `Choices` question type